docs/developer-guide/running-substreams.md

```diff
@@ -77,3 +77,46 @@ map_transfers: message "eth.erc721.v1.Transfers": {
 ```
 
 The example output shown above contains data for different transfers from data in the blockchain. These transfers can also be verified on [Etherscan](https://etherscan.io/tx/0xcfb197f62ec5c7f0e71a11ec0c4a0e394a3aa41db5386e85526f86c84b3f2796).
+
+## Development abd production mode
+
+### Development mode
+
+Production and development modes impact the execution of Substreams. Key aspects of execution include:
+
+- The time required to reach the first byte.
+- The speed that large ranges get executed.
+- The module logs and outputs that are sent back to the client.
+
+## Parallel execution in development versus production modes
+
+Differences between production and development modes include:
+
+- Forward parallel execution is enabled in production mode and disabled in development mode.
+- The time required to reach the first byte in development mode is faster than in production mode.
+
+Specific attributes of development modes include:
+
+- The client will receive all of the executed module's logs.
+- It's possible to request specific store snapshots in the execution tree.
+- Multiple module's output is possible.
+
+## Production and development parallel execution examples
+
+In most cases, you will run production mode, using a Substreams sink. Development mode is enabled by default in the CLI unless -p is specified.
+
+Examples: (given the dependencies: `[block] --> [map_pools] --> [store_pools] --> [map_transfers])`
+
+- Running the `substreams run substreams.yaml map_transfers` command `run` in development mode and only print the `map_transfers` module's outputs and logs.
```

Comment:
You used `run` but I meant it to be a word. My goal here is to make it clear to the user that he is running in development mode. 

```suggestion
- Running the `substreams run substreams.yaml map_transfers` command executes in development mode and only print the `map_transfers` module's outputs and logs.
```

The added part about development mode must also be added to the two other list item.

Reply:
Updates made and applied to all three list items. Please review for technical accuracy. Changes available in next push.